source/features/new-repo-disable-projects-and-wikis.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import {getRepoURL} from '../github-helpers';
+
+async function disableWikiAndProjects(): Promise<void | false> {
+	if (sessionStorage.rghNewRepo !== getRepoURL()) {
+		return false;
+	}
+
+	sessionStorage.removeItem('rghNewRepo');
+
+	api.v3(`repos/${getRepoURL()}`, {
+		method: 'PATCH',
+		body: {
+			has_projects: false,
+			has_wiki: false
+		}
+	});
+
+	const wiki = await elementReady('[data-tab-item="wiki-tab"]');
+	wiki?.closest('.d-flex')!.remove();
+	const projects = await elementReady('[data-content="Projects"]');
+	projects?.closest('.d-flex')!.remove();
+}
+
+function setStorage(): void | false {
+	if (!select<HTMLInputElement>('[name="rgh-disable-project"]')!.checked) {
+		return false;
+	}
+
+	const owner = select('#repository-owner')!.textContent!.trim();
+	const repository = select<HTMLInputElement>('.js-repo-name')!.value;
+	sessionStorage.rghNewRepo = owner + '/' + repository;
+}
+
+async function init(): Promise<void> {
+	await api.expectToken();
+
+	select.last('.js-repo-init-setting-container')!.after(
+		<div className="form-checkbox checked mt-0 mb-3">
+			<label>
+				<input checked type="checkbox" name="rgh-disable-project"/>
+				Remove Projects and Wiki`s
```

Comment:
Generally, [don't use `'s` to pluralize](https://www.dailywritingtips.com/when-to-form-a-plural-with-an-apostrophe/)
```suggestion
				Remove Projects and Wikis
```

and definitely don't use backticks as apostrophes 😄

Reply:
Lol I am so tired 

Will finish in the morning

Reply:
Also we should replace the word "remove" with "disable" everywhere.